factor/totp/lang/en/factor_totp.php

```diff
@@ -38,10 +42,14 @@
 $string['privacy:metadata'] = 'The TOTP Factor plugin does not store any personal data';
 $string['settings:secretlength'] = 'TOTP secret key length';
 $string['settings:secretlength_help'] = 'Generated TOTP secret key string length';
+$string['settings:window'] = 'TOTP verification window';
+$string['settings:window_help'] = 'How long each code is valid for. You can set this to a higher value as a workaround if your users device clocks are often slightly wrong.
+    Rounded down to the nearest 30 seconds, which is the time between new generated codes.';
 $string['setupfactor'] = 'TOTP Factor Setup';
 $string['setupfactor:scan'] = 'Enter secret or scan QR code';
 $string['setupfactor:key'] = 'Secret key: ';
 $string['verificationcode'] = 'Enter your 6 digit verification code';
 $string['verificationcode_help'] = 'Open your Authenticator app such as Google Authenticator and look for the 6 digit code which matches this site and username';
 $string['summarycondition'] = 'using a TOTP app';
 $string['factorsetup'] = 'Setup App';
+$string['systimeformat'] = '%D %T %Z';
```

Comment:
I think this is the neatest default:

%l:%M:%S%P %Z

5:17:36pm Australian Eastern Daylight Time